### DISCUSSION TITLE
Any way for frigate to take more frequent snapshots?

### DISCUSSION THREAD

**Discussion by ljmerza** [blakeblackshear/frigate#5913] [Ask A Question]
I use double-take to identify users to trigger automations but the problem I'm having is frigate for some cameras captures a "person" way too early so the image is far away and when double-take processes that image it doesn't get a high confidence vs a closer image. Is there a way for frigate to take more frequent snapshots so I can I get closer up images or somehow solve my problem? Thanks.

**Comment by NickM-27** [comment]
I would suggest using a zone and configuring doubletake to only look At snapshots when that zone is active. Then, you can setup and use latest for snapshots and that will work great (it does for me anyway)

**Reply by ljmerza** [reply]
That would work. Thanks
